app/helpers/cross_domain_analytics_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CrossDomainAnalyticsHelper
+  def cross_domain_tracker(slug)
+    {
+      'register-to-vote' => 'UA-23066786-5',
+      'accelerated-possession-eviction' => 'UA-37377084-12',
+      'renewtaxcredits' => 'UA-43414424-1', # HMRC, https://www.pivotaltracker.com/n/projects/1261204
```

Comment:
Having the link in the pull request is great, and Github makes it easy to go from a commit to a pull request. For example, https://github.com/alphagov/frontend/commit/141d9458b048ff886fe2c106134fd287601fb4f2 shows that this commit is part of #810.

Having the link in the commit is fine, it means we're not reliant on the Github history, though that's probably at least as persistent as our Pivotal Tracker project will be. Getting hold of that information is pretty straightforward using `git blame`.

Having the link as a comment in the code seems pointless, and suggests that to properly understand what's done here the reader should follow through that link.

I'm strongly in favour of dropping the link from the code.